src/main/resources/liquibase/202002270838-add-sla-column-default-values-and-indexes.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202002270838-1" author="mstead">
+        <comment>Create an index on the sla and product_id fields for snapshots.</comment>
+        <createIndex indexName="sla_product_id_idx" tableName="tally_snapshots"
+                     unique="false">
+            <column name="sla"/>
+            <column name="product_id" />
+        </createIndex>
+    </changeSet>
+
+    <changeSet id="202002270838-2" author="mstead">
+        <comment>Set the default sla to _ANY</comment>
+        <addDefaultValue tableName="tally_snapshots" columnName="sla" defaultValue="_ANY" />
+    </changeSet>
```

Comment:
@kahowell heh... Forgot this wasn't required due to the rule below. I think that the index could be moved over to your snapshot PR to keep it in one changeset file. I can remove this totally from here.

Reply:
:+1:  Will remove it.

Reply:
DONE